lib/fhir/resources/related-person.js

```diff
@@ -8,14 +8,15 @@
 'use strict'
```

Comment:
This resource could probably be deleted entirely.

Reply:
Yip i think we can remove this resource as Default should be able to handle it, This resource is however injected dirctly into doc-manifest. we will need to find a way to remove this resource, and have it use the Default one in the doc-manifest code

Reply:
Ah right, thanks for reminding me of this.